core/fuzz/README.md

```diff
@@ -0,0 +1,44 @@
+# Fuzz Test for OpenDAL
+
+fuzz test are used to test the robustness of the code. 
+
+## Setup
+
+To run the fuzz tests, you need to switch your rust version to nightly. You can do this by running the following command:
+
+```bash
+rustup override set nightly
```

Comment:
It's not a good idea to let our contributor use `rustup override set nightly`. I prefer to use `cargo +nightly xxx` instead.

Reply:
solved